app_vue/src/components/sensorRouteBlock.vue

```diff
@@ -0,0 +1,209 @@
+<script setup>
```

Comment:
sensorRouteBlock.vue is a new component which simply moves some of the code from the existing sensorSideBarRoutes.vue component